### PR TITLE
Add more test cases to the TOML file validation

### PR DESCRIPTION
**Pull request by vitchor**

### CODE REVIEW

File: cases/util/validateURL.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export const isURLvalid = async (str) => {
+    var pattern = new RegExp('^(https?:\\/\\/)?'+ // protocol
+      '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.)+[a-z]{2,}|'+ // domain name
+      '((\\d{1,3}\\.){3}\\d{1,3}))'+ // OR ip (v4) address
+      '(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*'+ // port and path
+      '(\\?[;&a-z\\d%_.~+=-]*)?'+ // query string
+      '(\\#[-a-z\\d_]*)?$','i'); // fragment locator
```
Comment (JakeUrban) [inline cases/util/validateURL.js]: I don't think the transfer server URL would have a fragment locator since its not a web page, right?

**Review by JakeUrban — APPROVED** [review]
Looks good other than my one comment. I'd suggest doing 1 PR for the rest of the TOML tests listed in the google sheet instead of test-by-test.

File: cases/util/validateURL.js (hunk 1 above)
Comment (msfeldstein) [reply cases/util/validateURL.js]: would it be easier to just do `new URL(str)` and let that throw an error if its not a valid url?

Comment (vitchor) [reply cases/util/validateURL.js]: I thought about making this as broad as possible to be reused in the future in other URL validations.

**Comment by vitchor** [comment]
Yup, I agree – I'll add the other TOML validations here. I just wanted to get some feedback on it, since it's my first validator test. :)

File: cases/util/validateURL.js (hunk 1 above)
Comment (msfeldstein) [reply cases/util/validateURL.js]: are there any URLs that wouldn't pass a `new URL()` test?  I'd rather be too strict and loosen it than too loose and allow broken things (and confusing code)

**Review by msfeldstein — APPROVED** [review]

**Pull request merged**